projects/core/package.json

```diff
@@ -23,8 +23,8 @@
   },
   "homepage": "https://github.com/moribvndvs/ng2-idle.git#readme",
   "peerDependencies": {
-    "@angular/common": "^10.0.0",
-    "@angular/core": "^10.0.0"
+    "@angular/common": ">=9.0.0",
```

Comment:
Ah, I see, this is how we loosen it up to support more than just 12. Is it good practice these days to put a ceiling on this and say `9.0.0 - 12.0.0`?

Reply:
Most of the libraries that I've used with a cap on the max version are _very_ actively maintained and are testing for compatibility against beta versions of Angular.

We might be better off leaving this uncapped and adding a warning to the readme if we discover an incompatibility with a future release of Angular.

Reply:
Leaving as-is, and noting intention to add a warning to the readme if an incompatibility with a future release is discovered.